_projects/improve_installation.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "This project is about improving the [bear installation tool](https://gitlab.com/coala/cib) and working around platform dependent requirements."
+desc: "This project is about improving the [bear installation tool](https://gitlab.com/coala/cib) and working around
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.